analysis/measures_definition_pf_codes_conditions.py

```diff
@@ -59,14 +59,21 @@
 imd = addresses.for_patient_on(INTERVAL.start_date).imd_rounded
 max_imd = 32844
 imd_quintile = case(
-    when((imd >= 0) & (imd < int(max_imd * 1 / 5))).then("1"),
+    when((imd >= 0) & (imd < int(max_imd * 1 / 5))).then("1 (Most Deprived)"),
     when(imd < int(max_imd * 2 / 5)).then("2"),
     when(imd < int(max_imd * 3 / 5)).then("3"),
     when(imd < int(max_imd * 4 / 5)).then("4"),
-    when(imd <= max_imd).then("5"),
+    when(imd <= max_imd).then("5 (Least Deprived)"),
     otherwise="Missing",
 )
 
+latest_region = registration.practice_nuts1_region_name
+
+latest_region = case(
+    when(latest_region.is_null()).then("Missing"),
+    otherwise=latest_region
+)
```

Comment:
this works, but I think we should use the `otherwise` argument for "Missing" and not for the actual values. we could also make this one call if you want

```suggestion
latest_region = case(
    when(
        registration.practice_nuts1_region_name.is_not_null()
    ).registration.practice_nuts1_region_name,
    otherwise="Missing",
)
```